webapp/public/index.html

```
@@ -0,0 +1,386 @@
+               viewBox="0 0 24 24" fill="none" stroke="currentColor"
+               stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round">
+            <line x1="18" y1="6"  x2="6"  y2="18"/>
+            <line x1="6"  y1="6"  x2="18" y2="18"/>
+          </svg>
+        </button>
+      </div>
+
+      <p class="pdf-panel-error" id="pdf-panel-error" style="display:none"></p>
+
+      <div class="pdf-panel-body">
+        <iframe id="pdf-iframe" src="" title="PDF viewer"
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ iframe ] must have a value.

(src-not-empty)